include/RequestLimiter.inc.php

```diff
@@ -188,11 +160,13 @@ public function finishConcurrent($bucket, $id) {
 		$key = 'crl:' . $bucket;
 		try {
 			$removed = $this->redis->zRem($key, $id);
-			if(!$removed) {
+			if (!$removed) {
 				Z_Core::logError('Failed to remove key Z_RequestLimiter::finishConcurrent');
 			}
 		} catch (Exception $e) {
-			Z_Core::logError('Redis error in Z_RequestLimiter::finishConcurrent: '.$e->getMessage());
+			Z_Core::logError('Redis error in Z_RequestLimiter::finishConcurrent: ' . $e->getMessage());
 		}
 	}
 }
+
+?>
```

Comment:
No `?>` necessary at end of file (and better to omit to avoid adding whitespace in some situations)